access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+:rhoas-version: 0.29.0
+
+// Other upstream project names
+:samples-git-repo: https://github.com/redhat-developer/app-services-guides
+
+//URL components for cross refs
+:base-url: https://github.com/redhat-developer/app-services-guides/blob/main/
+:base-url-cli: https://github.com/redhat-developer/app-services-cli/tree/main/docs/
+:getting-started-url: getting-started/README.adoc
+:kafka-bin-scripts-url: kafka-bin-scripts/README.adoc
+:kafkacat-url: kafkacat/README.adoc
+:quarkus-url: quarkus/README.adoc
+:rhoas-cli-url: rhoas-cli/README.adoc
+:rhoas-cli-ref-url: commands
+:topic-config-url: topic-configuration/README.adoc
+:consumer-config-url: consumer-configuration/README.adoc
+:service-binding-url: service-discovery/README.adoc
+:access-mgmt-url: access-mgmt/README.adoc
+
+////
+END GENERATED ATTRIBUTES
+////
+
+[id="chap-managing-access"]
+= Managing account access in {product-long}
+ifdef::context[:parent-context: {context}]
+:context: managing-access
+
+[IMPORTANT]
+====
+{product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
+====
+
+// Purpose statement for the assembly
+[role="_abstract"]
+As an owner of a Kafka instance in {product}, you can manage the level of access that other user accounts and service accounts have to your instance. You can allow or deny operations that specific accounts or that all accounts in your organization can perform for your instance or for certain resources in that instance, such as topics. You can also allow other users or service accounts to manage access to the resources in your instance for you.
+
+You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+//Additional line break to resolve mod docs generation error, not sure why. Leaving for now. (Stetson, 20 May 2021)
+
+[id="con-access-management_{context}"]
+== Access management in {product}
+
+[role="_abstract"]
+{product} uses Access Control Lists (ACLs) provided by Kafka that enable you to manage how other user accounts and service accounts are permitted to interact with the Kafka resources that you create. You can manage access for only the Kafka instances that you create or for instances that the owner has enabled you to access and alter.
+
+A resource in an ACL can be a Kafka instance, topic, consumer group, or producer transaction. A user account enables users in your organization to access your resources. A service account enables your application or tool to connect securely to your instance and access your resources.
+
+You can define ACLs for the following components:
```

Comment:
This statement sounds off to me. The components you describe are combined to create a single ACL, i.e. these are components or properties _of_ an ACL.

Reply:
Yep, good catch. Changed to "ACL permissions", which is one way around that issue. Been fighting a battle on not calling individual ACL permissions/entries just "ACLs" themselves, which is how others and even Kafka often refer to them.